client/signup/config/flows-pure.js

```diff
@@ -420,7 +420,7 @@ export function generateFlows( {
 			steps: [ 'store-address', 'business-info', 'confirm', 'transfer' ],
 			destination: '/',
 			description: 'Onboarding and installation flow for woocommerce on all plans.',
-			providesDependenciesInQuery: [ 'site' ],
+			providesDependenciesInQuery: [ 'site', 'back_to' ],
```

Comment:
```suggestion
			providesDependenciesInQuery: [ 'site', 'back_to' ],
			optionalDependenciesInQuery: [ 'back_to' ],
```

(See error when hitting `X` in the checkout step)

Reply:
Thanks! Fixed here: https://github.com/Automattic/wp-calypso/pull/60892/commits/22515490ed20baec47b50a81cbcd289edd38f0f0